sql/core/src/main/scala/org/apache/spark/sql/execution/LocalTableScanExec.scala

```diff
@@ -80,8 +80,7 @@ case class LocalTableScanExec(
   // Input is already UnsafeRows.
   override protected val createUnsafeProjection: Boolean = false
 
-  // Do not codegen when there is no parent - to support the fast driver-local collect/take paths.
-  override def supportCodegen: Boolean = (parent != null)
+  override def supportCodegen: Boolean = true
```

Comment:
We don't need this cuz CodegenSupport already sets true.

Reply:
Ah, exactly.